tests/ui/admin_flow.py

```diff
@@ -146,17 +146,21 @@ def change_admin_password(self, old_password, new_password,
         EC.invisibility_of_element_located(((
             UfOPageLayout.DROPDOWN_MENU_SPINNER))))
 
-  def remove_test_admin(self, email, server_url, should_raise_exception=True):
+  def remove_test_admin(self, email, server_url, should_raise_exception=True,
+                        should_navigate_to_landing=True):
     """Remove a test admin account using a form post (the only way currently).
 
     Args:
       email: A string for the email of the test admin to remove.
       server_url: The base url portion of the landing page.
       should_raise_exception: True to raise an exception if the admin is not
                               found.
+      should_navigate_to_landing: True remove the admin using the landing page,
```

Comment:
nit: is this description, right?  Should it really say, "True, go to the landing page."
Actually this comment can also be:
"Boolean of whether to go to the landing page or not."